fixed main_split_image.dart's error

rushikesh127001 opened the pull request:
Fixed this error on build:" error: The function expression type 'Future<Codec> Function(Uint8List, {int cacheHeight, int cacheWidth})' isn't of type 'Future<Codec> Function(Uint8List, {bool allowUpscaling, int cacheHeight, int cacheWidth})'. This means its parameter or return type doesn't match what is expected. Consider changing parameter type(s) or the returned type(s). (invalid_cast_function_expr at [flutter_samples] lib/split_image/main_split_image.dart:20)"

It was missing the bool allowupscaling  parameter. I have added that, and now its working just fine.

![Screenshot from 2020-10-02 14-02-04](https://user-images.githubusercontent.com/50454029/94904067-5adf2880-04b8-11eb-81f7-05456ba4769f.png)

diegoveloper commented:
What's your flutter version?

On Fri, Oct 2, 2020, 3:36 AM rushikesh127001 <notifications@github.com>
wrote:

> Fixed this error on build:" error: The function expression type 'Future
> Function(Uint8List, {int cacheHeight, int cacheWidth})' isn't of type
> 'Future Function(Uint8List, {bool allowUpscaling, int cacheHeight, int
> cacheWidth})'. This means its parameter or return type doesn't match what
> is expected. Consider changing parameter type(s) or the returned type(s).
> (invalid_cast_function_expr at [flutter_samples]
> lib/split_image/main_split_image.dart:20)"
>
> It was missing the bool allowupscaling parameter. I have added that, and
> now its working just fine.
>
> [image: Screenshot from 2020-10-02 14-02-04]
> <https://user-images.githubusercontent.com/50454029/94904067-5adf2880-04b8-11eb-81f7-05456ba4769f.png>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/diegoveloper/flutter-samples/pull/24
> Commit Summary
>
>    - fixed main_split_image.dart's error
>
> File Changes
>
>    - *M* lib/split_image/main_split_image.dart
>    <https://github.com/diegoveloper/flutter-samples/pull/24/files#diff-3aeb596394b2ec091a38b4d61bc994ce>
>    (2)
>
> Patch Links:
>
>    - https://github.com/diegoveloper/flutter-samples/pull/24.patch
>    - https://github.com/diegoveloper/flutter-samples/pull/24.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/diegoveloper/flutter-samples/pull/24>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABFL3UFJCBVNNCG4XPGRVTTSIWGI5ANCNFSM4SBNRZYQ>
> .
>

rushikesh127001 commented:
> What's your flutter version?
> […](#)
> On Fri, Oct 2, 2020, 3:36 AM rushikesh127001 ***@***.***> wrote: Fixed this error on build:" error: The function expression type 'Future Function(Uint8List, {int cacheHeight, int cacheWidth})' isn't of type 'Future Function(Uint8List, {bool allowUpscaling, int cacheHeight, int cacheWidth})'. This means its parameter or return type doesn't match what is expected. Consider changing parameter type(s) or the returned type(s). (invalid_cast_function_expr at [flutter_samples] lib/split_image/main_split_image.dart:20)" It was missing the bool allowupscaling parameter. I have added that, and now its working just fine. [image: Screenshot from 2020-10-02 14-02-04] <https://user-images.githubusercontent.com/50454029/94904067-5adf2880-04b8-11eb-81f7-05456ba4769f.png> ------------------------------ You can view, comment on, or merge this pull request online at: #24 Commit Summary - fixed main_split_image.dart's error File Changes - *M* lib/split_image/main_split_image.dart <https://github.com/diegoveloper/flutter-samples/pull/24/files#diff-3aeb596394b2ec091a38b4d61bc994ce> (2) Patch Links: - https://github.com/diegoveloper/flutter-samples/pull/24.patch - https://github.com/diegoveloper/flutter-samples/pull/24.diff — You are receiving this because you are subscribed to this thread. Reply to this email directly, view it on GitHub <#24>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/ABFL3UFJCBVNNCG4XPGRVTTSIWGI5ANCNFSM4SBNRZYQ> .

my flutter version is "Flutter (Channel stable, 1.20.3, on Linux, locale en_IN.UTF-8)"
this is the latest stable version i guess

diegoveloper commented:
great, thanks for the contribution!
